### DISCUSSION TITLE
Adiciona metadados de municípios | Add municipalities metadata

### DISCUSSION THREAD

**Discussion by Luisa-Coelho** [okfn-brasil/querido-diario-comunidade#9] [Ideas]
### Português
_Metadados de municípios podem ajudar pesquisadores e formuladores de políticas a analisar tendências, clusters e também entender padrões na administração pública._

**O que pode ser incluído:**
- Tamanho da população;
- Coordenadas geográficas;
- PIB/capita;
- Produção agrícola

### English
_Metadata from municipalities may help researchers and policy makers analyze trends, clusters and also understand patterns in the public administration._

**What can be included:**
- Population size;
- Geographical coordinates;
- GDP/capita;
- Agriculture production.

**Comment by ogecece** [comment]
@Luisa-Coelho no caso a ideia que você havia comentado seria a de possibiltar filtros de busca baseados nesses metadados, né?

**Reply by Luisa-Coelho** [reply]
Isso! 👍

**Comment by trevineju** [comment]
Outros metadados que podem ser usados em filtro de busca podem ser:
- Por Estado (UF)
- Regiões Administrativas do país (Norte, Sul, Nordeste...)
- Agrupamentos Geográficos (Estados e Municípios da Amazônia Legal, da Mata Atlântica)

**Reply by trevineju** [reply]
Issue no repositório da API para desenvolver o filtro por UF: https://github.com/okfn-brasil/querido-diario-api/issues/54

**Comment by trevineju** [comment]
Sugestão de fonte de metadados dada por Begnini na reunião de pessoas mantenedoras (Abril/22): as bases do IBGE
- https://www.ibge.gov.br/estatisticas/downloads-estatisticas.html
- https://ftp.ibge.gov.br/Perfil_Municipios/2020/Base_de_Dados/Base_MUNIC_2020.xlsx

_obs.: O Querido Diário já usa os id's de municípios na base do censo._

**Comment by trevineju** [comment]
Em um contexto de integração de bases de dados de diversos projetos internos da OKBR, outros filtros para busca poderiam experimentar correlações com a Serenata de Amor, Parlametria, Queremos Saber
